h/static/scripts/directive/via-dialog.coffee

```diff
@@ -0,0 +1,27 @@
+###*
+# @ngdoc directive
+# @name viaLinkDialog
+# @restrict A
+# @description The dialog that generates a via link to the page h is currently
+# loaded on.
+###
+module.exports = ['$timeout', 'crossframe', ($timeout, crossframe) ->
+    viaUrl = 'https://via.hypothes.is/h/'
```

Comment:
:+1: Also (somewhat related), the `/h/` is no longer needed. https://github.com/hypothesis/via/issues/1